qemu/tests/virtio_fs_rlimit_nofile.py

```
@@ -0,0 +1,71 @@
+        finally:
+            session.close()
+    elif rlimit_nofile == "1024":
+        error_context.context(
+            "Starting virtiofsd with rlimit-nofile=%s" % rlimit_nofile,
+            test.log.info)
+        vm = env.get_vm(params.get("main_vm"))
+        vm.verify_alive()
+        session = vm.wait_for_login()
+        session.close()
+    else:
+        test.fail("Invalid rlimit-nofile: %s" % rlimit_nofile)
```

Ruff (0.12.2)

No newline at end of file

Add trailing newline

(W292)